book/src/http_consensus.md

```diff
@@ -0,0 +1,182 @@
+# Lighthouse REST API: `/consensus`
+
+The `/consensus` endpoints provide information on results of the proof-of-stake
+voting process used for finality/justification under Casper FFG.
+
+## Endpoints
+
+HTTP Path | Description |
+| --- | -- |
+[`/consensus/global_votes`](#consensusglobal_votes) | A global vote count for a given epoch.
+[`/consensus/individual_votes`](#consensusindividual_votes) | A per-validator breakdown of votes in a given epoch.
+
+## `/consensus/global_votes`
+
+Returns a global count of votes for some given `epoch`. The results are included
+both for the current and previous (`epoch - 1`) epochs since both are required
+by the beacon node whilst performing per-epoch-processing.
+
+Generally, you should consider the "current" values to be incomplete and the
+"previous" values to be final. This is because validators can continue to
+include attestations from the _current_ epoch in the _next_ epoch, however this
+is not the case for attestations from the _previous_ epoch.
```

Comment:
I was confused by this paragraph until I looked at the code and saw that the `epoch` being requested must have already come to an end, i.e. you can't request data midway through the current wall-clock epoch. Perhaps this API would be better returning just one set of balances, which it derives from the `previous_` fields of the following epoch's `ValidatorStatuses` struct. So if the current wall-clock epoch is `N`, you can use this function to request finalized voting information for epochs up to `N - 2`. That has the disadvantage of not being able to see what happened in the previous wall-clock epoch... but with the current design we already can't see what's happening in the current wall-clock epoch. Perhaps there could be two API endpoints? One for "finalized" data from all epochs up to `N - 2` and another for "live" info from the current and previous epochs?

Reply:
As discussed with @michaelsproul, we'll leave this as-is for now.